apps/docs/src/app/platform/api-files.ts

```diff
@@ -20,6 +20,12 @@ export const API_FILES = {
     searchField: ['SearchFieldComponent'],
     select: ['SelectPlatformComponent'],
     splitMenuButton: ['SplitMenuButtonComponent'],
+    table: [
+        'TableComponent',
+        'TableToolbarComponent',
+        'TableToolbarActionsComponent',
+        'TableColumnComponent'
+    ],
```

Comment:
We need to add these directives as well:
``` 
  FdpTableCell,
  FdpCellDef,
  FdpTableHeader,
  FdpHeaderCellDef
```

Reply:
Thanks, you absolutely right